# What: Putting a manually typed version in setup.cfg, as it wouldnt resolve automatically.
# Author: Denis Kristak (Inuits)diff -ruN tenacity-8.2.3_orig/setup.cfg tenacity-8.2.3/setup.cfg
# Updated for v9.0.0 by maxim-masterov (SURF)
# Updated with magic string to avoid having to update the patch for every version by Samuel Moors (Vrije Universiteit Brussel)
diff -Nru tenacity-9.0.0.orig/setup.cfg tenacity-9.0.0/setup.cfg
--- tenacity-9.0.0.orig/setup.cfg	2024-10-10 16:50:28.669538307 +0200
+++ tenacity-9.0.0/setup.cfg	2024-10-10 16:50:54.881500726 +0200
@@ -1,6 +1,7 @@
 [metadata]
 name = tenacity
 license = Apache 2.0
+version = 'EB_TENACITY_VERSION'
 url = https://github.com/jd/tenacity
 summary = Retry code until it succeeds
 long_description = Tenacity is a general-purpose retrying library to simplify the task of adding retry behavior to just about anything.
